services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx

```diff
@@ -33,6 +34,7 @@ const AppLayoutView = ({ sideBarContent, displaySideBar, children }) => {
           <Content style={styles.content}>{children}</Content>
         </Layout>
       </Layout>
+      <Footer />
     </Layout>
```

Comment:
According to https://ant.design/components/layout/
```suggestion
          <Content style={styles.content}>{children}</Content>
          <Footer />
        </Layout>
      </Layout>
    </Layout>
```